tests/console/sle15_workarounds.pm

```diff
@@ -25,10 +25,11 @@ sub run {
     return unless sle_version_at_least('15');
     select_console 'root-console';
     # Kernel devel packages are not in the dev tools module, so add standard repos
-    record_soft_failure('bsc#1053222');    # Once bug is resolved, this code can be removed
+    record_soft_failure('bsc#1054062');    # Once bug is resolved, this code can be removed
```

Comment:
Don't you think it is something too obvious to mention it ? 
For me it looks the same as put in every line something like 
`#this line suppose to work actually ` :) 
But if you insists I am ok to merge it like this